git-modified-since

```diff
@@ -30,9 +30,14 @@ if [ $# -ge 1 ]; then
 fi
 
 make_relative () {
+    realpath='realpath'
+    if ! type realpath >/dev/null 2>&1; then
+        # Mac/BSD ports system installs GNU coreutils with 'g' prefixes
+        realpath='grealpath'
+    fi
```

Comment:
Rather than assuming that `grealpath` exists if `realpath` does not, I'd prefer to do another `type` test there first before assigning it to the realpath variable, and failing with a helpful error message if not.

Reply:
That was my instinct, too, but I didn't want to scare you off by multiplying the size of this svelte little function threefold, and then copying it in triplicate. `:)`

Are we reaching a point where you would feel another layer of abstraction is necessary (e.g., `git-relative-path`) to avoid code duplication, or is this amount of duplication below your threshold?

I am happy to handle it either way.

Reply:
I would _love_ to make this a separate utility. `git-relative-path` seems useful as a CLI tool / building block 👍